runners/local-artifact-service-java/build.gradle

```diff
@@ -23,6 +23,7 @@ description = "Apache Beam :: Runners :: Java Local Artifact Service"
 
 dependencies {
   compile library.java.guava
+  compile project(path: ":beam-runners-parent:beam-java-fn-execution", configuration: "shadow")
```

Comment:
this should be shadow

Compile means it is shaded away, see build_rules.gradle#applyJavaNature

Reply:
Done.